KFC.Red.Server/KFC.Red.DBate.WebAPI/Controllers/QuestionController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using KFC.Red.ManagerLayer.QuestionManagement;
+using KFC.Red.DataAccessLayer.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace KFC.Red.DBate.WebAPI.Controllers
+{
+    public class QuestionManagementRequest
+    {
+        [HttpBindRequired]
+        public Question Question { get; set; }
+
+        [HttpBindRequired]
+        public int QuestionID { get; set; }
+
+        [HttpBindRequired]
+        public string QuestionString { get; set; }
+    }
+
+    public class QuestionController : ApiController
+    {
+        // http GET/POST/PUT/DELETE methods based on add/update/delete questions
+        // see user controller in SSO
+        [HttpPost]
+        [Route("api/question/add")]
+        public IHttpActionResult AddQuestion([FromBody] QuestionManagementRequest Request)
+        {
+            QuestionManager questionM = new QuestionManager();
```

Comment:
You have to start the QuestionManager to do the logic such as adding/deleting/updating question.
Example: questionM.addQuestion(question);